src/fetch_albums_lambda/fetch_album.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import base64
+import boto3
+from datetime import datetime
+import os
+
+
+GET_TOKEN_URL = "https://accounts.spotify.com/api/token"
+BASIC_API_URL = "https://api.spotify.com"
+
+api_client_id = os.environ['API_CLIENT_ID']
+api_client_secret = os.environ['API_CLIENT_SECRET']
+
+album_table_name = os.environ['ALBUM_TABLE_NAME']
+
+dynamodb_resource = boto3.resource('dynamodb')
+album_table = dynamodb_resource.Table(album_table_name)
+
+
+def get_access_token_from_api():
+    auth = 'Basic ' + base64.b64encode((api_client_id + ':' + api_client_secret).encode()).decode()
+    response_token = requests.post(GET_TOKEN_URL, headers={'Authorization' : auth, 'Content-Type' : 'application/x-www-form-urlencoded'}, data = {'grant_type' : 'client_credentials'})
+    return response_token.json()
+
+
+def get_artist_id_from_api(credentials, artist_name):
+    auth = 'Bearer ' + credentials['access_token']
+    response_artist = requests.get(BASIC_API_URL + '/v1/search?q=' + artist_name + '&type=artist&market=DE&limit=1&offset=0', headers={'Authorization' : auth})
+    id_of_artist = (response_artist.json())['artists']['items'][0]['id']
+    return id_of_artist
```

Comment:
```suggestion
    return (response_artist.json())['artists']['items'][0]['id']
```

you can directly return the result